addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -193,8 +176,42 @@ function factory(dependencies) {
          *
          * @param {MouseEvent} ev
          */
-        onClickStartAMeetingButton(ev) {
-            this.createMeetingChannel();
+        async onClickStartAMeetingButton(ev) {
+            const meetingChannel = await this.messaging.models['mail.thread'].createGroupChat({
+                default_display_mode: 'video_full_screen',
+                partners_to: [this.messaging.currentPartner.id],
+            });
+            await meetingChannel.open();
+            if (!meetingChannel.exists()) {
+                return;
+            }
+            await meetingChannel.toggleCall({ startWithVideo: true });
+            if (!this.exists()) {
+                return;
+            }
+            if (!this.threadView) {
+                return;
+            }
+            if (!this.threadView.channelInvitationForm) {
+                this.threadView.update({
+                    channelInvitationForm: insert({
+                        threadView: this.threadView,
+                    }),
+                });
+            }
+            this.update({
+                invitePopover: insert({
+                    anchorRef: this.startAMeetingButtonRef,
+                    discussOwner: this,
+                    position: 'bottom',
+                    target: replace(
+                        this.threadView.channelInvitationForm,
+                    ),
+                    targetComponentName: 'ChannelInvitationForm',
```

Comment:
all of those could be compute actually because it's not actually generic and identifying fields have discuss clearly

Reply:
actually should use the one from thread view top bar directly, no need for a discuss one